### PR TITLE
AddCommand sequence diagram

### LINKED ISSUE #101: Developer Guide: Food Implementation

### LINKED ISSUE #124: Update Developers Guide
1. Add sequence diagrams
2. Update sections
3. Create a contents page
4. Overall cosmetics

### PR DESCRIPTION
**Pull request by Vinci-Hu**

### CODE REVIEW

File: docs/diagrams/AddSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    opt equals newFoodName
+        Checker --> Fridge: true
+    end
+    return false
+end
+
+alt Foodname is unique (add new food))
+    Fridge --> AddCommand: food added to fridge
+
+else all parameters except quantity consists with record (add to existing)
```
Comment (Leeyp) [inline docs/diagrams/AddSequenceDiagram.puml]: Perhaps rephrase this into something that sounds nicer?

File: docs/diagrams/AddSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    Fridge -> Checker: checker.isParamIdentical()
+    activate Checker
+    return boolean
+    note right
+    If params do not consist,
```
Comment (Leeyp) [inline docs/diagrams/AddSequenceDiagram.puml]: It's not very clear what this means to 'consist'.

**Review by Leeyp — CHANGES_REQUESTED** [review]
Sequence diagram generally looks good, but some changes for clarification are needed!

Also, the "Implementation" for Add command paragraph in the DG needs to be updated.

**Review by SimJJ96 — APPROVED** [review]
LGTM

**Review by Leeyp — APPROVED** [review]
LGTM

**Review by kwokyto — APPROVED** [review]
LGTM

**Pull request merged**